### ISSUE TITLE
CTRXplorer freezes when viewing folder with hundreds of files

### ISSUE THREAD

**Issue by onmyouji** [d0k3/CTRXplorer#22]
I'm not sure if this is a bug or known limitation, but when I try to view a folder inside my SD card, it freezes. 

There are about 300 files inside the folder, the total size is 13MB (the files are ExtData backup of Fire Emblem Echoes, exported using JKSV).

System: Old 3DS XL
Firmware: 11.5
Luma3DS v8.1.1

**Comment by d0k3** [comment]
By freeze, do you mean it just takes long for teh folder contents to show, or do you mean CTRXplorer crashes?

**Comment by onmyouji** [comment]
I'm not really sure because once CTRXplorer became unresponsive, I immediately reboot my 3DS (using the reboot feature from Luma). 

I waited for about a minute before rebooting. I didn't wait longer because I was worried it might corrupt my SD card or something, sorry.

::EDIT::
I tried to delete the folder but it freeze again, so I had to delete it from my computer.

**Comment by Masamune3210** [comment]
I have noticed the same issue, if the folder has over 50 items in it, it takes a while to show the contents each time it is opened. The actual amount of time depends on how fast the sd card is and how deep in the folder structure the folder in question is

**Comment by KevinBlue18** [comment]
I agree with them. I have a badge folder with over 700+ badges, and copying over to the default badge folders freezes.

**Comment by gingerbeardman** [comment]
I also have this issue. 

GodMode9 is quick to display the same directory. 
RetroArch file browser is very quick to display the same directory. 

New 3DS, latest Luma and A9H and system software
